test/unit/vdom/properties.js

```diff
@@ -117,4 +117,36 @@ describe('vdom/properties', () => {
       );
     });
   });
+
+  it('#876 - Apply properties and attributes in a polyfill environment', (done) => {
+    const Elem1 = define('x-test', {
+      render (elem) {
+        return h(Elem2, { foo: true, bar: true, baz: true });
```

Comment:
The camelCasing / dash-casing is handled by the property lifecycles when `{ attribute: true }` is specified.